modules/nw-multus-bridge-object.adoc

```diff
@@ -43,42 +61,63 @@ a value is not specified, the `default` namespace is used.
+the CNO configuration.
 
-<2> `bridge`: Specify the name of the virtual bridge to use. If the bridge
+<2> Specify the name of the virtual bridge to use. If the bridge
 interface does not exist on the host, it is created. The default value is
 `cni0`.
 
-<3> `ipam`: Specify a configuration object for the ipam CNI plug-in. The plug-in
+<3> Specify a configuration object for the ipam CNI plug-in. The plug-in
 manages IP address assignment for the network attachment definition.
 
-<4> `ipMasq`: Set to `true` to enable IP masquerading for traffic that leaves the
+<4> Set to `true` to enable IP masquerading for traffic that leaves the
 virtual network. The source IP address for all traffic is rewritten to the
 bridge's IP address. If the bridge does not have an IP address, this setting has
 no effect. The default value is `false`.
 
-<5> `isGateway`: Set to `true` to assign an IP address to the bridge. The
+<5> Set to `true` to assign an IP address to the bridge. The
 default value is `false`.
 
-<6> `isDefaultGateway`: Set to `true` to configure the bridge as the default
+<6> Set to `true` to configure the bridge as the default
 gateway for the virtual network. The default value is `false`. If
 `isDefaultGateway` is set to `true`, then `isGateway` is also set to `true`
 automatically. 
 
-<7> `forceAddress`: Set to `true` to allow assignment of a previously assigned
+<7> Set to `true` to allow assignment of a previously assigned
 IP address to the virtual bridge. When set to `false`, if an IPv4 address or an
 IPv6 address from overlapping subsets is assigned to the virtual bridge, an
 error occurs. The default value is `false`.
 
-<8> `hairpinMode`: Set to `true` to allow the virtual bridge to send an ethernet
+<8> Set to `true` to allow the virtual bridge to send an ethernet
 frame back through the virtual port it was received on. This mode is also known
 as _reflective relay_. The default value is `false`.
 
-<9> `promiscMode`: Set to `true` to enable promiscuous mode on the bridge. The
+<9> Set to `true` to enable promiscuous mode on the bridge. The
 default value is `false`.
 
-<10> `vlan`: Specify a virtual LAN (VLAN) tag as an integer value. By default,
+<10> Specify a virtual LAN (VLAN) tag as an integer value. By default,
 no VLAN tag is assigned.
 
-<11> `mtu`: Set the maximum transmission unit (MTU) to the specified value. The
-default value is automatically chosen by the kernel.
+<11> Set the maximum transmission unit (MTU) to the specified value. The
+default value is automatically set by the kernel.
+
+[id="nw-multus-bridge-config-example_{context}"]
+== bridge configuration example
+
+The following example configures an additional network named `bridge-net`:
+
+[source,yaml]
+----
+name: bridge-net
+type: Raw
+rawCNIConfig: '{ <1>
+  "cniVersion": "0.3.1",
+  "type": "bridge",
+  "master": "eth1",
+  "ipam": {
+    "type": "dhcp"
+    }
+}'
+----
+<1> The CNI configuration object is specified as a YAML string.
```

Comment:
Probably my ignorance. You said above the CNI plug-in configuration is JSON format??
Can you get a more robust example with more of the parameters from the tamplate?

Reply:
Replied on a later comment.